schemas/dto/initJobExecutionsRqDto.json

```diff
@@ -0,0 +1,16 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "description": "Request to initialize JobExecution entities",
+  "additionalProperties": false,
+  "properties": {
+    "files": {
+      "description": "Information about files to upload",
+      "type": "array",
+      "items": {
+        "type": "object",
+        "$ref": "fileDto.json"
```

Comment:
"../mod-source-record-manager/raml/dto/fileDto.json"

Reply:
fileDto.json located in the same package, schemas from mod-source-record-manager will be removed